modules/ROOT/pages/manage-client-apps-latest-task.adoc

```diff
@@ -3,12 +3,27 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In Exchange, you can make information about APIs you develop in Anypoint Platform publicly available for applications that want to consume the API. You can limit access to certain users.
+In API Manager, you can grant access to applications that want to consume your APIs by creating instance Contracts. +
 
-The applications, to which you have granted access, are listed on API Manager > Contracts. As Organization Administrator of the master organization, you can manage contracts registered to access your API.
+To view your API Instance contracts:
 
-image::api-manager-client-app.png[contracts]
+. In API Manager, click *API Administration*.
+. Click the API Name of your desired API instance.
+. Click the API Version of your selected API instance. +
+This opens the configuration page of your selected API instance.
 
-Select an app name to view app information and access the following controls for removing the app (revoking access), resetting the client ID and client secret, and adding or removing app owners:
+image::api-manager-administrator.png[api-manager-administrator]
```

Comment:
This image shows the landing page of API Manager, shouldn't it be below the first step that states `In API Manager, click *API Administration*.`?

Reply:
-DS_Store files were not uploaded in purpose. Sometimes these are in the folder and uploaded automatically.
-Rendered file: Yes, it's OK since Github will not display the images.
-Updated the API Administrator image position after step 1.